libraries/package-deps-hash/src/getRepoState.ts

```diff
@@ -290,7 +290,11 @@ async function spawnGitAsync(
     pipeline(stdin, proc.stdin!, (err) => {});
   }
 
-  const [status] = await once(proc, 'exit');
+  const [[status]] = await Promise.all([
+    once(proc, 'exit'),
+    once(proc.stdout!, 'close'),
+    once(proc.stderr!, 'close')
+  ]);
```

Comment:
It should suffice to wait for the `'close'` event on `proc`: https://nodejs.org/api/child_process.html#event-close

Reply:
I've tried that and it didn't seem to help, really, the issue was still happening. I'll double check.

Reply:
Well, I stand corrected, double checking seems to have been worth it :-) 
I swear I tried this, because I found the exact same documentation paragraph as well, but I must've done something incorrectly, because it didn't help. I've tried it now and it solves the issue. 🤷‍♂️